spec/polaris-management-service.yml

```diff
@@ -862,6 +862,7 @@ components:
           type: string
           enum:
             - ICEBERG_REST
+            - HADOOP
```

Comment:
Since the `HadoopCatalog` is highly unsafe if the set of people who have `CATALOG_CREATE` privileges isn't exactly equal to the set of people with superuser privileges to run the entire Catalog service itself, before we introduce this one even as an option we should probably add a `ALLOWED_CATALOG_CONNECTION_TYPES` or `SUPPORTED_CATALOG_CONNECTION_TYPES` analogous to `SUPPORTED_CATALOG_STORAGE_TYPES`.

Reply:
Great idea, added